contribs/gmf/src/controllers/abstractmobile.js

```diff
@@ -31,16 +33,31 @@ goog.require('ol.style.Style');
 gmfModule.constant('isMobile', true);
 
 
+gmfModule.constant(
+    'treeUrl',
+    'https://geomapfish-demo.camptocamp.net/2.0/wsgi/themes?' +
+        'version=2&background=background');
```

Comment:
We need to think about that @sbrunner. Those constants will be generated on load in the mako template (within c2cgeoportal).
Should we rather put those definitions in the `index.html` file of gmf contrib as well ?